tools/travis/deploy_cos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+HOMEDIR="$ROOTDIR/.."
+WHISKDIR="$HOMEDIR/openwhisk"
+# Set Environment
+export OPENWHISK_HOME=${OPENWHISK_HOME:=$WHISKDIR}
```

Comment:
Since OPENWHISK_HOME does not exist above this line wouldn't this always just == WHISKDIR?

Reply:
This allows to run the script locally in my laptop where I always have OPENWHISK_HOME define